src/prototype_room_creepbuilder.js

```diff
@@ -67,9 +67,9 @@ Room.prototype.spawnCheckForCreate = function(creepsConfig) {
   let storages;
   let energyNeeded;
   let unit;
-
-  if (this.memory.queue.length > 0 && (creepsConfig.length === 0 || creepsConfig[0] != 'harvester')) {
-    let room = this;
+  let room = this;
```

Comment:
this is already available for entire function :) i personally prefer to use this when we can.
for use in others functions like sortByDistance you can move them as room method and so aply them to this ( plus, i don't remember why but i believe this method already exist somewhere).